go-controller/pkg/cluster/management-port_linux.go

```diff
@@ -68,9 +74,14 @@ func CreateManagementPort(nodeName string, localSubnet *net.IPNet, clusterSubnet
 	}
 
 	// Create a route for the services subnet.
-	_, _, err = util.RunIP("route", "add", config.Kubernetes.ServiceCIDR, "via", routerIP)
+	_, stderr, err := util.RunIP("route", "add", config.Kubernetes.ServiceCIDR, "via", routerIP)
```

Comment:
This change shouldn't be needed; the service subnet isn't obviously located on this interface so there's no way the route could have been automatically added

Reply:
Maybe this was added as precautionary after adding the one above?  not sure ... we can try with just the first one and see if it still works?

Reply:
or how about leaving it and just making it a debug message?  Ignoring this seems pretty harmless ...